great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -37,14 +37,22 @@ class NumericMetricRangeMultiBatchParameterBuilder(MultiBatchParameterBuilder):
     ranges, which will incorporate the requirements, imposed by the configured false_positive_rate tolerances.
     """
 
+    RECOGNIZED_TRUNCATE_DISTRIBUTION_KEYS: set = {
+        "lower_bound",
+        "upper_bound",
+    }
+
     def __init__(
         self,
         parameter_name: str,
         metric_name: str,
```

Comment:
Given that this is the NumericRangeMultiBatchParameterBuilder, would it make sense for it to automatically build a ParameterContainer with the calculated min and max (instead of needing to specify `column.min` and `column.max`)? Is there a chance that a user would use this ParameterBuilder to get a metric other than min or max here? If not, it seems excessive to require a separate ParameterBuilder for min and max.